src/cart/crt_rpc.c

```diff
@@ -208,6 +208,21 @@ static struct crt_proto_rpc_format crt_fi_rpcs[] = {
 
 #undef X
 
+#define X(a, b, c, d, e) case a: return #a;
+
+char
+*crt_opc_to_str(crt_opcode_t opc)
+{
+	switch (opc) {
+	CRT_INTERNAL_RPCS_LIST
+	CRT_FI_RPCS_LIST
+	default:
+		return "External rpc";
```

Comment:
I think that we should print the binary opcode instead in this case.

Reply:
I think you mean hex opcode; if so - in the RPC_ERROR where this helper function is used we print hex opcode followed by string for example (opc=0xff020010 (CRT_OPC_CTL_LS)) . For DAOS/external RPCs it will print something like (opc=0x200010 (External rpc)) - would you rather it have print hex value twice?